sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/EventGridModelFactory.cs

```diff
@@ -1,16 +1,141 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+using System.Collections.Generic;
 using System.ComponentModel;
 using System.Text.Json;
-using Azure.Core;
 using Azure.Messaging.EventGrid.SystemEvents;
 
 namespace Azure.Messaging.EventGrid
 {
 #pragma warning disable CA1054 // URI-like parameters should not be strings
     public static partial class EventGridModelFactory
     {
+        /// <summary> Initializes new instance of MediaJobError class. </summary>
```

Comment:
Can we hide all of these with EditorBrowsableState.Never?

Reply:
Sure

Reply:
Fixed